Policies/Subscription/Azure_Subscription_Config_MDC_Setup_SecurityContacts/Security - Subscription - ConfigMDCSecurityContacts.json

```diff
@@ -0,0 +1,76 @@
+{
+    "mode": "All",
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
+          "description": "The effect determines what happens when the policy rule is evaluated to match"
+        },
+        "allowedValues": [
+          "Audit",
+          "Disabled",
+          "Deny"
+        ],
+        "defaultValue": "Audit"
+      }
+    },
+    "policyRule": {
+      "if": {
```

Comment:
Which resource type will be flagged by this policyy

Reply:
Updated, we are targeting subscription here.